src/pages/docs/octopus-rest-api/octopus.server.exe-command-line/configure.md

```diff
@@ -108,7 +108,7 @@ Where [<options>] is any of:
       --trustedProxies=VALUE Comma-separated list of IP addresses of trusted
                                proxies
       --webTrustedRedirectUrls=VALUE
-                             Comma-seperated list of URLs that are trusted
+                             Comma-separated list of URLs that are trusted
```

Comment:
This is probably based on an actual program's buggy output, so please consider this as a bug report.